core/trino-main/src/main/java/io/trino/connector/system/TableCommentSystemTable.java

```diff
@@ -152,6 +152,10 @@ public RecordCursor cursor(ConnectorTransactionHandle transactionHandle, Connect
         return metadata.getRedirectionAwareTableHandle(session, tableName).getTableHandle()
                 .map(handle -> metadata.getTableMetadata(session, handle))
                 .map(metadata -> metadata.getMetadata().getComment())
-                .get();
+                .orElseGet(() -> {
+                    // A previously listed table might have been dropped concurrently
```

Comment:
`getComment` is already wrapped in `try catch`, why we need this?

Reply:
That catches TrinoException, the Optional.get call throws RuntimeException when it is empty